src/clj/game/cards/agendas.clj

```diff
@@ -175,7 +174,12 @@
                    (do (system-msg state :runner "chooses to take 2 tags from Armed Intimidation")
                        (gain-tags state :runner eid 2 {:card card}))
                    ; else
-                   (effect-completed state side eid)))}})
+                   (effect-completed state side eid)))})]
+    {:on-score
+     {:interactive (req true)
```

Comment:
I think the only change that's necessary is putting `:interactive true` in the `:on-score` ability. Moving the main ability to a function doesn't actually change anything given that there's no recursion.